openfisca_core/entities.py

```diff
@@ -53,6 +53,23 @@ def __call__(self, variable_name, period = None, options = [], **parameters):
         else:
             return self.simulation.calculate(variable_name, period, **parameters)
 
+    # JSON
```

Comment:
What is the aim of this comment? If it only says `JSON` on a method called `to_json`, better remove it altogether.